frontend/src/components/adjudications/AdjudicationTableRow.jsx

```diff
@@ -1,12 +1,72 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import TableCell from '@material-ui/core/TableCell';
+import Input from "@material-ui/core/Input";
 import TableRow from '@material-ui/core/TableRow';
 import Button from '@material-ui/core/Button';
+import IconButton from "@material-ui/core/IconButton";
+import EditIcon from "@material-ui/icons/EditOutlined";
+import DoneIcon from "@material-ui/icons/DoneAllTwoTone";
+import RevertIcon from "@material-ui/icons/NotInterestedOutlined";
 import { Link } from 'react-router-dom';
+import { updateAdjudications, getAdjudicationByPerformanceAndJudge } from '../../api/AdjudicationAPI';
 
 class AdjudicationTableRow extends React.Component {
-  state = {};
+  state = {adjudication: {}, isEditMode: false, technicalMark: {}, artisticMark: {} };
+  componentDidMount() {
+    getAdjudicationByPerformanceAndJudge(this.props.id, 1) //hradcoded tablet_id
+    .then(({data}) => {
+      this.setState({ adjudication: data});
```

Comment:
these two can be combined into one line, calling setState twice in a line may make the screen flicker

Reply:
also please destructure state (https://stackoverflow.com/questions/52055987/destructuring-state-props-in-react)